requirements.txt

```diff
@@ -3,6 +3,9 @@ validators>=0.18.2,<0.20.0
 tqdm>=4.59.0,<5.0.0
 authlib>=1.1.0
 
+numpy
+torch
+tensorflow
```

Comment:
hu? that is a bit heavy?

Reply:
I added this only for development, it is used to test if we can extract vectors from different libraries.

We can omit this and test only with built-in `list`. What do you think?

Reply:
I removed them in favour for Mocking them.

Reply:
nice idea :)